tests/bsim/babblekit/include/babblekit/testcase.h

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "bs_tracing.h"
+#include "bs_types.h"
+#include "bstests.h"
+
+extern enum bst_result_t bst_result;
+
+/*
+ * @brief Initialize the test framework
+ *
+ * Call this at the start of the test entry point.
+ *
+ * @param timeout_us Timeout (in microseconds) until the timeout callback
+ * @param message String to print to console after init
+ *
+ */
+#define TEST_START(timeout_us, message)                                                            \
+	do {                                                                                       \
+		bst_ticker_set_next_tick_absolute((bs_time_t)timeout_us);                          \
+		bst_result = In_progress;                                                          \
```

Comment:
Sidecomment: This is already done in the bst framework, so it is redundant to do it again here, but it does no harm.